sequencing-server/src/app.ts

```diff
@@ -36,8 +40,17 @@ app.use(bodyParser.json({ limit: '100mb' }));
 
 DbExpansion.init();
 export const db = DbExpansion.getDb();
+export let graphqlClient = new GraphQLClient(getEnv().MERLIN_GRAPHQL_URL, {
+  headers: { 'x-hasura-admin-secret': getEnv().HASURA_GRAPHQL_ADMIN_SECRET },
+});
+export const piscina = new Piscina({
+  filename: new URL('worker.js', import.meta.url).pathname,
+  minThreads: parseInt(getEnv().SEQUENCING_WORKER_NUM),
+  resourceLimits: { maxOldGenerationSizeMb: parseInt(getEnv().SEQUENCING_MAX_WORKER_HEAP_MB) },
```

Comment:
Default these env variables to a reasonable value in case they're not set?

Reply:
Checkout `getEnv()` in `env.ts` If there isn't one provided in the environment from the docker-compose it will use the defaults.